src/report.cc

```diff
@@ -70,6 +70,10 @@ void report_t::normalize_options(const string& verb)
     if (HANDLED(pager_) && ! isatty(STDOUT_FILENO))
       HANDLER(pager_).off();
   }
+#else
+  if (HANDLED(force_color)) {
+    HANDLER(color).on("?normalize");
+  }
```

Comment:
At first glance this appears unrelated to emscripten, what is the motivation for this change?

Reply:
No, this does not seem relevant anymore.

(A bit of context: I wrongly commented out the `check_function_exists` checks on `HAVE_ISATTY` `HAVE_IOCTL` when I first encountered the `ioctl_tiocgwinsz` bug mentioned above, which somehow got me to believe Emscripten does not provide an `isatty` implementation. And I made the above changes to pass https://github.com/ledger/ledger/blob/master/test/baseline/opt-force-color.test .)